### ISSUE TITLE
Cannot update: certificate has expired

### ISSUE THREAD

**Issue by SCFred28** [OpenDroneMap/WebODM#1064]
### What's your browser and operating system?

Chrome 94 on Windows 10

### What is the problem?

Cannot update: certificate has expired

![image](https://user-images.githubusercontent.com/83292330/135648283-35199839-4cff-4b79-944a-112f6e31f682.png)

### What should be the expected behavior?

Continue to receive update.

### How can we reproduce this? (

With WebODM 1.9.7. build 32 click on the "Tools" Menu then "Check for Updates"

**Comment by pierotofy** [comment]
Mm, strange! If you open https://webodm.org/test/ do you get a certificate error?

![image](https://user-images.githubusercontent.com/1951843/135651120-6204a907-0d0d-4fef-89d7-a06223f5e29d.png)

**Comment by pierotofy** [comment]
Latest version is build `32` though, so good news is you're on the latest version.

**Comment by SCFred28** [comment]
I give a try for a fresh reinstall.

------ Message d'origine ------
De: "Piero Toffanin" ***@***.***>
À: "OpenDroneMap/WebODM" ***@***.***>
Cc : "SCFred28" ***@***.***>; "Author" 
***@***.***>
Envoyé : 2021-10-01 12:03:16
Objet : Re: [OpenDroneMap/WebODM] Cannot update: certificate has expired 
(#1064)

**Comment by SCFred28** [comment]
Not working either. Do I need to renew the license? Is their any 
expiring?

------ Message d'origine ------
De:
À: "OpenDroneMap/WebODM" 
***@***.***>; 
"OpenDroneMap/WebODM" ***@***.***>
Cc : "Author" ***@***.***>
Envoyé : 2021-10-01 12:58:07
Objet : Re[2]: [OpenDroneMap/WebODM] Cannot update: certificate has 
expired (#1064)

>I give a try for a fresh reinstall.
>
>------ Message d'origine ------
>De: "Piero Toffanin" ***@***.***>
>À: "OpenDroneMap/WebODM" ***@***.***>
>Cc : "SCFred28" ***@***.***>; "Author" 
***@***.***>

**Comment by pierotofy** [comment]
No, no need to renew; it's a problem with our SSL certificate. We need to find a fix for it.

**Comment by pierotofy** [comment]
https://www.zdnet.com/article/fortinet-shopify-others-report-issues-after-root-ca-certificate-from-lets-encrypt-expires/

**Comment by SCFred28** [comment]
Apparently you have to switch to the latest available ISRG Root X1 valid until 2035-06-04 (intermediate R3)https://letsencrypt.org/certificates/Télécharger Outlook pour AndroidDe : Piero Toffanin ***@***.***>Envoyé : vendredi 1 octobre 2021 16 h 01À : OpenDroneMap/WebODMCc : SCFred28; AuthorObjet : Re: [OpenDroneMap/WebODM] Cannot update: certificate has expired (#1064)
https://www.zdnet.com/article/fortinet-shopify-others-report-issues-after-root-ca-certificate-from-lets-encrypt-expires/

**Comment by SCFred28** [comment]
Help thread about this problem:https://community.letsencrypt.org/t/help-thread-for-dst-root-ca-x3-expiration-september-2021/149190Télécharger Outlook pour AndroidDe : Frédéric Bouchard ***@***.***>Envoyé : vendredi 1 octobre 2021 16 h 31À : OpenDroneMap/WebODM; OpenDroneMap/WebODMCc : AuthorObjet : Re: [OpenDroneMap/WebODM] Cannot update: certificate has expired (#1064)Apparently you have to switch to the latest available ISRG Root X1 valid until 2035-06-04 (intermediate R3)https://letsencrypt.org/certificates/Télécharger Outlook pour AndroidDe : Piero Toffanin ***@***.***>Envoyé : vendredi 1 octobre 2021 16 h 01À : OpenDroneMap/WebODMCc : SCFred28; AuthorObjet : Re: [OpenDroneMap/WebODM] Cannot update: certificate has expired (#1064)
https://www.zdnet.com/article/fortinet-shopify-others-report-issues-after-root-ca-certificate-from-lets-encrypt-expires/

**Comment by jakevis** [comment]
+1 - have a repo here as well if you need anything
